helm/alfresco-content-services/charts/alfresco-search/Chart.yaml

```diff
@@ -13,11 +13,11 @@ keywords:
 name: alfresco-search
 sources:
   - https://github.com/Alfresco/acs-deployment
-version: 1.2.0
+version: 1.2.0-SNAPSHOT
```

Comment:
dependencies should be increased to next snapshot (1.3.0-SNAPSHOT) - check if the docs are explicit about this